store/postgres/migrations/2018-07-10-062730_create_revert_procedures/up.sql

```diff
@@ -252,24 +252,7 @@ $$
 DECLARE
     event_row RECORD;
     entity_row RECORD;
-    subgraph_ptr_hash VARCHAR;
```

Comment:
Can't we easily update this revert procedure by removing it and adding the new version in a new migration?

Reply:
Yup! Should I write a proper migration to avoid needing to dropdb? (This part is pretty easy... the hard part is translating data from the `subgraphs` and `subgraph_deployments` tables into subgraph entities to keep the subgraph block pointer intact, etc)